back/src/composables/useUtils.ts

```diff
@@ -58,12 +58,45 @@ const sendResponse = (response: Response, data: any, statusCode = 200) => {
   });
 };
 
+const toDbFormat = (body: any) => {
+  const convertedBody: any = {};
+
+  for (const key in body) {
+    if (Object.prototype.hasOwnProperty.call(body, key)) {
+      const snakeCaseKey = key.replace(
+        /[A-Z]/g,
+        (match) => `_${match.toLowerCase()}`
+      );
+      convertedBody[snakeCaseKey] = body[key];
+    }
+  }
+
+  return convertedBody;
+};
+
+const fromDbFormat = (body: any) => {
```

Comment:
Looks like it's an object type, not *any* no?

Reply:
True, gonna change that